clients/client/src/main/java/org/projectnessie/client/StreamingUtil.java

```diff
@@ -55,16 +55,16 @@ private StreamingUtil() {
    * @param builderCustomizer a Function that takes and returns {@link GetAllReferencesBuilder} that
    *     is passed to the caller of this method. It allows the caller to customize the
    *     GetAllReferencesBuilder with additional parameters (e.g.: filter).
-   * @param pageSizeHint - the page size hint passed to the Nessie server. Undefined, if {@code
-   *     Optional.empty()}.
+   * @param maxRecords - a maximum number of records in the stream. If it is {@code
+   *     Optional.empty()} the stream will be unbounded.
    * @return stream of {@link Reference} objects
    * @deprecated Use {@link GetAllReferencesBuilder#stream(OptionalInt)}
    */
   @Deprecated
   public static Stream<Reference> getAllReferencesStream(
       @NotNull NessieApiV1 api,
       @NotNull Function<GetAllReferencesBuilder, GetAllReferencesBuilder> builderCustomizer,
-      @NotNull OptionalInt pageSizeHint)
+      @NotNull OptionalInt maxRecords)
```

Comment:
This is potentially used by external code... Do we really have to change this parameter's semantics?

Reply:
Where's that change??